lib/src/model.dart

```diff
@@ -28,8 +28,27 @@ import 'model_utils.dart';
 import 'package_meta.dart' show PackageMeta, FileContents;
 import 'utils.dart' show stripComments;
 
+Map<String, Map<String, List<Map<String, dynamic>>>> __crossdartJson;
+
 final Map<Class, List<Class>> _implementors = new Map();
 
+Map<String, Map<String, List<Map<String, dynamic>>>> get _crossdartJson {
```

Comment:
Not a comment for this PR, but this nesting of generics is too deep. I don't know what they're maps of list of strings to -